app/assets/javascripts/controllers/container_deployment/deploy-provider-controller.js

```diff
@@ -0,0 +1,254 @@
+miqHttpInject(angular.module('miq.containers.providersModule', ['ui.bootstrap', 'patternfly', 'miq.dialogs', 'miq.wizard', 'ManageIQ'])).controller('containers.deployProviderController',
+  ['$rootScope', '$scope', '$timeout', 'miqService',
+  function($rootScope, $scope, $timeout, miqService) {
+    'use strict';
+
+    $scope.showDeploymentWizard = false;
+    ManageIQ.angular.scope = $scope;
+    $scope.data = {};
+    $scope.nodeData = {
+      allNodes: [],
+      filteredNodes: [],
+      providerVMs: [],
+      newVMs: [],
+      userDefinedVMs: []
+    };
+
+    var initializeDeploymentWizard = function () {
+      $scope.deployProviderReady = false;
+      $scope.deployComplete = false;
+      $scope.deployInProgress = false;
+      $scope.deploySuccess = false;
+      $scope.deployFailed = false;
+
+      $scope.data = {
+        providerName: '',
+        providerType: 'openshiftOrigin',
+        provisionOn: 'existingVms',
+        masterCount: 0,
+        nodeCount: 0,
+        infraNodeCount: 0,
+        cdnConfigType: 'satellite',
+        authentication: {
+          mode: 'all'
+        }
+      };
+
+      $timeout(function() {
```

Comment:
because it takes time for the api request to return with the needed info and then load it to the relevant variable and fields.
if this is removed on most tries the substeps don't load fast enough.